Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,7 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.filter((index1, index2, index3) => index3.indexOf(index1) === index2);
```

Comment:
While the code passes the test, this line is difficult to grasp as the names `index1`, `index2` and `index3` do not convey anything useful. In fact, only one of them is truly an index. If you look up the documentation of `filter` (google 'mdn filter') you will find the argument names `element`, `index` and `array`. You don't really need the third argument as you can already access the array through the `arr` variable.

Your code becomes easier to understand if you pick appropriate names:

```js
return arr.filter((element, index) => arr.indexOf(element) === index);
```

Reply:
My bad...It's done.
Thanks!!